.github/workflows/ci.yml

```diff
@@ -41,35 +41,19 @@ jobs:
       - name: create instrumentation coverage
         uses: ReactiveCircus/android-emulator-runner@v2
         env:
-          GRADLE_OPTS: "-Dorg.gradle.internal.http.connectionTimeout=60000 -Dorg.gradle.internal.http.socketTimeout=60000 -Dorg.gradle.internal.network.retry.max.attempts=6 -Dorg.gradle.internal.network.retry.initial.backOff=2000"
-        if: ${{ matrix.api-level != 33 }}
+         GRADLE_OPTS: "-Dorg.gradle.internal.http.connectionTimeout=60000 -Dorg.gradle.internal.http.socketTimeout=60000 -Dorg.gradle.internal.network.retry.max.attempts=6 -Dorg.gradle.internal.network.retry.initial.backOff=2000"
         with:
-          api-level: ${{ matrix.api-level }}
-          target: default
-          arch: x86_64
-          profile: pixel_2
-          ram-size: '4096M'
-          disk-size: '14G'
-          sdcard-path-or-size: '1000M'
-          disable-animations: false
-          script: bash contrib/instrumentation.sh
+         api-level: ${{ matrix.api-level }}
+         target: ${{ matrix.api-level != 33 && 'default' || 'google_apis' }}
+         arch: x86_64
+         profile: pixel_2
+         ram-size: ${{ matrix.api-level != 33 && '4096M' || '4096M' }}
```

Comment:
@gouri-panda Since `ram-size` is the same for both API levels, we should get rid of this condition here.